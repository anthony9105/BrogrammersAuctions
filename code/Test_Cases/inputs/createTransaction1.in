create
newUser12345   
SS
abc
login newUser12345 abc 
bid racecar currentSeller21
150
logout