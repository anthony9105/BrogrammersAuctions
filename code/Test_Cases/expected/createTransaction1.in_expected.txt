Enter new username
Enter type of user (admin (AA), full-standard (FS), buy-standard (BS), or sell-standard (SS))
Enter password (over 1 character, less than 15 characters)
Success.  New user added.
Login successful
Error.  You do not have the privilege to do this transaction.
logout successful